packages/transformers/js/src/visitors/fs.js

```diff
@@ -64,10 +69,16 @@ export default ({
 
         let replacementNode;
         if (Buffer.isBuffer(res)) {
-          replacementNode = bufferTemplate({
-            CONTENT: t.stringLiteral(res.toString('base64')),
-            ENC: t.stringLiteral('base64'),
-          });
+          let str = t.stringLiteral(res.toString('base64'));
+          if (asset.env.isNode()) {
+            replacementNode = bufferTemplate({
+              CONTENT: str,
+            });
+          } else {
+            replacementNode = atobTemplate({
```

Comment:
Won't this change the semantics of the code? `atob` returns a string, not a Buffer.

Reply:
🙈